examples/About.ipynb

```diff
@@ -8,18 +8,39 @@
     "[holoviz.org](https://holoviz.org) website shows how to use Param\n",
     "together with other libraries to solve complex problems, with detailed\n",
     "tutorials and examples. Example projects using Param are at\n",
-    "[examples.pyviz.org](https://examples.pyviz.org).\n",
+    "[examples.pyviz.org](https://examples.pyviz.org). and you can compare Param to other available tools at [pyviz.org](https://pyviz.org).\n",
+    "\n",
+    "Param is completely open source, available under a BSD license freely for both commercial and non-commercial use. Param was originally developed with the support of [Anaconda Inc](https://anaconda.com), and is now maintained by Anaconda developers and community contributors.\n",
```

Comment:
Please link "BSD license" to the license file on GitHub so that it's clear what the master declaration is.

Reply:
Updated